app/helpers/enum_i18n_helper.rb

```diff
@@ -0,0 +1,15 @@
+module EnumI18nHelper
+  def enum_options_for_select(class_name, enum)
+    class_name.send(enum.to_s.pluralize).map do |key, _|
+      [enum_i18n(class_name, enum, key), key]
+    end
+  end
+  
+  def enum_l(model, enum)
```

Comment:
```suggestion
  def enum_l model, enum
```
những chỗ tương tự bỏ () đi nhé